File: baekjoon/p01120/Main.java
package baekjoon.p01120;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next(), sc.next()));
    }

    private static int solve(String a, String b) {
        int ret = 987654321;
        for (int i = 0; i <= b.length() - a.length(); i++) {
            int diff = getDiff(a, b.substring(i, i + a.length()));
            ret = Math.min(ret, diff);
        }

        return ret;
    }

    private static int getDiff(String a, String b) {
        int ret = 0;
        int len = a.length();

        for (int i = 0; i < len; i++) {
            if (a.charAt(i) != b.charAt(i)) {
                ret++;
            }
        }

        return ret;
    }

}

File: baekjoon/p01120/Main.kt
package baekjoon.p01120

fun main() {
    val inputs = readLine()!!.split(" ")
    println(solve(inputs[0], inputs[1]))
}

private fun solve(a: String, b: String): Int {
    var ret = Int.MAX_VALUE
    for (i in 0..b.length - a.length) {
        val diff = (0 until a.length).count { a[it] != b[i + it] }
        ret = minOf(ret, diff)
    }

    return ret
}